src/session/appRouter/getPermission.js

```diff
@@ -1,10 +1,21 @@
-import { getClaim } from "./getClaim";
+import {createKindeServerClient} from '@kinde-oss/kinde-typescript-sdk';
+import {cookies} from 'next/headers';
+import {config} from '../../config/index';
+import {sessionManager} from '../../session/sessionManager';
 
 export const getPermission = (key) => {
-  const orgCode = getClaim("org_code");
-  const permissions = getClaim("permissions") || [];
-  return {
-    isGranted: permissions.some((p) => p === key),
-    orgCode,
-  };
+  const kindeClient = createKindeServerClient(
+    config.grantType,
+    config.clientOptions
+  );
```

Comment:
@peterphanouvong you should be able to just create one instance of the kindeClient and share it across all helpers - maybe check in with @coel